R/extractManualRecode.R

```diff
@@ -5,7 +5,8 @@
 #'
 #' @examples # tbd
 extractManualRecode <- function(recodedList) {
-  if (!is.data.frame(recodedList)) stop("'df' must be a data.frame.")
+
```

Comment:
Is there a specific reason for this empty line at the start of a function definition? Does `lintr` prefer this formatting?

Reply:
No, it's a personal preference, but I'm not consistent with it anyways.